src/server-extension/resolvers/VideosResolver/index.ts

```diff
@@ -116,8 +116,15 @@ export class VideosResolver {
           : ''),
       ''
     )
+    const viewsOrder =
+      args.orderBy.find((orderBy) => orderBy.startsWith('viewsNum_'))?.split('_')?.[1] || 'DESC'
+
     idsQuerySql = overrideClause(idsQuerySql, 'GROUP BY', '"video"."id"')
-    idsQuerySql = overrideClause(idsQuerySql, 'ORDER BY', 'COUNT("video_view_event"."id") DESC')
+    idsQuerySql = overrideClause(
+      idsQuerySql,
+      'ORDER BY',
+      `COUNT("video_view_event"."id") ${viewsOrder}`
```

Comment:
I think the `orderBy` should not affect the id query itself. The query is called `mostViewedVideosConnection`, which already indicates that the search is always done among `x` most viewed videos. If you change the order here it becomes `leastViewedVideos`

Reply:
good point, I removed the change